install/infra/modules/gke/variables.tf

```diff
@@ -60,3 +60,23 @@ variable "credentials" {
   description = "Path to the JSON file storing Google service account credentials"
   default     = ""
 }
+
+variable "domain_name" {
+  description = "Domain name register with Cloud DNS, leave empty if you want to manage it yourself"
+  default     = null
+}
+
+variable "enable_external_database" {
+  default     = true
+  description = "Set this to false to avoid creating an RDS database to use with Gitpod instead of inclsuter mysql"
+}
+
+variable "enable_external_storage" {
+  default     = true
+  description = "Set this to false to avoid creating an s3 storage to use with Gitpod instead of incluster minio"
+}
+
+variable "enable_external_registry" {
+  default     = true
+  description = "Set this to false to create an AWS ECR registry to use with Gitpod(Not officially supported)"
+}
```

Comment:
**Fixup:** We have some lingering EKS references, WDYT of the following?

```suggestion
variable "enable_external_database" {
  default     = true
  description = "Create an external Cloud DNS instance for Gitpod"
}

variable "enable_external_storage" {
  default     = true
  description = "Create a GCS bucket for Gitpod object storage"
}

variable "enable_external_registry" {
  default     = true
  description = "Create a Google Container Registry for Gitpod workspace images"
}
```

Reply:
weird I cannot commit this suggestion since they say it is outdated. Applied manually.